UnityProject/Assets/Scripts/Objects/Gateway/StationGateway.cs

```diff
@@ -146,14 +149,14 @@ protected virtual void UpdateMe()
 				if (APCPoweredDevice.IsOn(CurrentState) == false) return;
 
 				timeElapsedServer += Time.deltaTime;
-				if (timeElapsedServer > DetectionTime && isOn)
+				if (timeElapsedServer > DetectionTime && isOn && isOnCooldown == false)
 				{
 					DetectPlayer();
 					timeElapsedServer = 0;
 				}
 
 				timeElapsedServerSound += Time.deltaTime;
-				if (timeElapsedServerSound > SoundLength && isOn)
+				if (timeElapsedServerSound > SoundLength && isOn && isOnCooldown == false)
 				{
 					DetectPlayer();
```

Comment:
Think the actual issue is this.

Not sure why I made it do a detect on sound play time as well as detection time, think it was the second thing I coded for the project :p

Reply:
Removed it but my fix is still required for the other issue mentioned.